pkgs/by-name/gi/gitbutler/package.nix

```
@@ -57,15 +65,32 @@ rustPlatform.buildRustPackage rec {
 
     substituteInPlace apps/desktop/src/lib/backend/tauri.ts \
       --replace-fail 'checkUpdate = check;' 'checkUpdate = () => null;'
+
+    cp ${./Cargo.lock} Cargo.lock
   '';
 
-  useFetchCargoVendor = true;
-  cargoHash = "sha256-wzSRUZeB5f9Z/D+Sa5Nl77jh7GDnnUehcmwanPcaSKM=";
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "gix-0.72.1" = "sha256-1v7/3F8ItBG940tei3YOhFpSK4Fl2BaCriczMGNBVPs=";
+      "tauri-plugin-trafficlights-positioner-1.0.1" =
+        "sha256-4VjU7Vf+zznk7qqyaum+bLuslVNxeGSoxDPz8QQp0pk=";
+    };
+  };
 
   pnpmDeps = pnpm_9.fetchDeps {
-    inherit pname version src;
+    inherit
+      pname
+      version
+      src
+      patches
+      ;
     fetcherVersion = 1;
```

Can you try updating to the latest [fetcherVersion](https://nixos.org/manual/nixpkgs/stable/#javascript-pnpm-fetcherVersion)? You also need to regenerate the hash. It hopefully fixes the hash mismatch error that happened to a few people

```suggestion
    fetcherVersion = 2;
```